components/schemas/StorageIntegration.yaml

```diff
@@ -0,0 +1,15 @@
+type: array
+items:
+  - type: object
+    properties:
+      name:
+        type: string
+        description: Name/path of the bucket
+      protocol:
+        type: string
+        enum: [s3, gs]
+        description: Protocol used by the integration
```

Comment:
```suggestion
      protocol:
        $ref: './StorageProtocol.yaml'
```

Reply:
Can't do that, because this is a schema, and the referenced part is a parameter.
I changed it and the linter started screaming at me that protocols aren't supposed to have names

Reply:
I tried to do this, but since this is a schema, and "StorageProtocol" is a reference it kind of screamed at me. Am I doing something wrong?